camisole/languages/rust.py

```diff
@@ -3,5 +3,5 @@
 
 class Rust(Lang):
     source_ext = '.rs'
-    compiler = Program('rustc', opts=['-W', 'warnings', '-O'])
+    compiler = Program('rustc', opts=['-W', 'warnings', '-C opt-level=3'])
```

Comment:
If this is an `argv` list, these probably need to be split into two strings : `'-C', 'opt-level=3'`

Reply:
That's right, arguments should be correct now.